tests/storage/upgrade/conftest.py

```diff
@@ -68,54 +68,58 @@ def skip_if_not_override_cdiconfig_scratch_space(override_cdiconfig_scratch_spec
 
 
 @pytest.fixture(scope="session")
-def cirros_vm_for_upgrade_a(
+def rhel_vm_for_upgrade_a(
     upgrade_namespace_scope_session,
     admin_client,
     storage_class_for_snapshot,
     cluster_common_node_cpu,
+    rhel10_data_source_scope_session,
 ):
     with create_vm_for_snapshot_upgrade_tests(
         vm_name="snapshot-upgrade-a",
         namespace=upgrade_namespace_scope_session.name,
         client=admin_client,
         storage_class_for_snapshot=storage_class_for_snapshot,
         cpu_model=cluster_common_node_cpu,
```

Comment:
should be a modern_cpu? 
IIRC, someone said that latest rhel may not work on old cpu models